unime/src/routes/welcome/password/confirm/+page.svelte

```diff
@@ -12,12 +14,23 @@
   import Smiley from '~icons/ph/smiley';
   import SmileySad from '~icons/ph/smiley-sad';
 
-  let passwordsEqual: boolean | undefined;
+  // 3 states: true (match), false (mismatch), undefined (I don't know).
```

Comment:
I would update `"I don't know"` with `"unknown"` or `"not checked yet"`.

Reply:
Comment changed.